project/Common.scala

```diff
@@ -53,22 +50,13 @@ object Common extends AutoPlugin {
         }
       }
     },
-    // Play 3 not supported in 2.11/2.12
-    scalaVersion := "2.13.11",
-    crossScalaVersions := {
-      val baseVersions = if (playVer.value startsWith "2.") {
-        Seq("2.11.12",
-          "2.12.17")
-      } else {
-        Seq.empty
-      }
-
-      baseVersions ++
-      Seq(
-        scalaVersion.value,
-        "3.2.2"
-      )
-    },
+    scalaVersion := "2.12.17",
+    crossScalaVersions := Seq(
+      "2.11.12",
+      scalaVersion.value,
+      "2.13.11",
+      "3.3.3"
```

Comment:
Driver/BSON lib is in 3.2.2 due to Scala regression, so it's better to keep it

Reply:
Ok since it was 3.3.3 in the ci test i will align all version to 3.2.2

Reply:
Done